workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Versions.targets

```diff
@@ -49,7 +49,7 @@ Copyright (c) Samsung All rights reserved.
   <ItemGroup>
     <KnownFrameworkReference
       Include="Samsung.Tizen"
-      TargetFramework="net6.0-tizen$(_DefaultTargetPlatformVersion)"
+      TargetFramework="$(TargetFramework)"
```

Comment:
need to consider the use of implcit tpm version. e.g., `net6.0-tizen`.

Reply:
Thanks for pointing that out! I added the code supporting that.